src/sentry/static/sentry/app/types/hooks.ts

```diff
@@ -360,6 +360,8 @@ type IntegrationFeatureListProps = FeatureGateSharedProps & {
   provider: Pick<IntegrationProvider, 'key'>;
 };
 
+export type FeatureListComponent = React.ComponentType<IntegrationFeatureListProps>;
```

Comment:
-1 for exporting like this. I would prefer just doing the type lookup in the `Hook` type.

Have you seen this

https://github.com/getsentry/sentry/blob/338ced4ddd722c6906d56eecd9f5d266c7dd5978/src/sentry/static/sentry/app/components/modals/integrationDetailsModal.tsx#L28-L50